lockbox-ios/Store/AutoLockStore.swift

```diff
@@ -86,7 +94,7 @@ extension AutoLockStore {
 
     private func setTimer(seconds: Int) {
         let timerValue = self.userDefaults.double(forKey: SettingKey.autoLockTimerDate.rawValue)
-        if timerValue != 0 && timerValue > Date().timeIntervalSince1970 {
+        if timerValue != 0 && timerValue < Date().timeIntervalSince1970 {
```

Comment:
this makes me wonder how this was ever working ...

Reply:
It seems like this should trip up the unit tests. It seems like this will create an invalid timer.

Reply:
It worked because we were always calling `stopTimer()` right before `setTimer()`, so `timerValue` was _always_ 0.

With this new change, the timer is invalidated, but the `timerValue` in `userDefaults` is not zeroed.

Top tip: never use `>`.